### PR TITLE
Sway Ref: Style guide unused variables

### LINKED ISSUE #4656: Sway Ref: Style guide unused variables

### PR DESCRIPTION
**Pull request by Braqzen**
## Description

Closes #4656 

Added `getters` to forc workspace because it wasn't in there for some reason.

## Checklist

- [x] I have linked to any relevant issues.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation where relevant (API docs, the reference, and the Sway book).
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added (or requested a maintainer to add) the necessary `Breaking*` or `New Feature` labels where relevant.
- [x] I have done my best to ensure that my PR adheres to [the Fuel Labs Code Review Standards](https://github.com/FuelLabs/rfcs/blob/master/text/code-standards/external-contributors.md).
- [x] I have requested a review from the relevant team or maintainers.

### CODE REVIEW

File: docs/reference/src/documentation/language/style-guide/unused-variables.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Unused Variables
+
+A good practice is naming variables appropriately; however, variables may be unused at times such as the `timestamp` from the `call()`.
+
+```sway
+{{#include ../../../code/language/style-guide/unused_variables/src/lib.sw:named_unused_variable}}
+```
+
+When a variable is unused we can discard its value by assigning it to `_`.
```
Comment (IGI-111) [inline docs/reference/src/documentation/language/style-guide/unused-variables.md:9]: This should mention you can prefix with _ and conserve the naming for context, here as `_timestamp`.

Comment (Braqzen) [reply docs/reference/src/documentation/language/style-guide/unused-variables.md:9]: I have considered that and based on your comment in #4655 I've opted to exclude the name. I'll adjust it in a bit

File: docs/reference/src/code/Forc.lock [hunk 2]
```diff
@@ -62,184 +62,203 @@ source = 'member'
...
 
 [[package]]
 name = 'simple_script'
 source = 'member'
-dependencies = ['std']
+dependencies = ['std path+from-root-0F28276288D2432C']
+
+[[package]]
+name = 'std'
+source = 'git+https://github.com/fuellabs/sway?tag=v0.40.1#48104d0bde0d343154a5bc39a310092532883235'
```
[the remark this answers is not in the document]
Comment (Braqzen) [reply docs/reference/src/code/Forc.lock:190]: Unsure, my process is to change into the root of the repo and `cargo build` and then I use that local `forc` in the workspace to build all the projects to update the lock file.

**Review by K1-R1 — APPROVED** [review]
New docs content looks good, but the git dependency path in docs/reference/src/code/Forc.lock is odd

**Comment by anton-trunov** [comment]
LGTM too, modulo the git dep thing

**Review by anton-trunov — APPROVED** [review]

**Review by K1-R1 — APPROVED** [review]

**Pull request merged**